### PR TITLE
Add process merge image

### PR DESCRIPTION
**Pull request by m-chaturvedi**

### CODE REVIEW

**Comment by mattdawkins** [comment]
Build is failing with this due to a new parameter to convert to opencv image container (need to specify the color type) though I'm going to try and add a default argument on another branch, you still probably want that in here so it works with the current master

**Comment by m-chaturvedi** [comment]
Oh, yeah, I forgot to pull it before pushing in the new changes.  I was working from home yesterday.

File: sprokit/processes/kwiver_type_traits.h [hunk 1]
```diff
@@ -115,6 +115,8 @@ create_port_trait( gsd, gsd, "GSD for image in meters per pixel." );
 create_port_trait( image, image, "Single frame image." );
 create_port_trait( left_image, image, "Single frame left image." );
 create_port_trait( right_image, image, "Single frame right image." );
+create_port_trait( image1, image, "Single frame first image." );
+create_port_trait( image2, image, "Single frame second image." );
```
Comment (linus-sherrill) [inline sprokit/processes/kwiver_type_traits.h]: These two port traits are algorithm specific enough to be moved to the process source.

Comment (m-chaturvedi) [reply sprokit/processes/kwiver_type_traits.h]: Moved.

**Comment by dstoup** [comment]
Jenkins test this please

**Comment by m-chaturvedi** [comment]
The Mac build had failed with the same test failures [before my PR.](http://proteus:8080/job/KwiverMacPR/556/)

**Comment by dstoup** [comment]
> The Mac build had failed with the same test failures [before my PR.](http://proteus:8080/job/KwiverMacPR/556/)


Nothing to worry about there, those are just failing test which are long standing (and now fixed on master)!

**Comment by linus-sherrill** [comment]
If this is going to merge up to three images with optional inputs, it would be better to convert to a dynamic input port approach, as is done in the new detected object merge process. It will give more flexibility in port naming and future expansion.

**Comment by mattdawkins** [comment]
We'll worry about the 3rd port latter lets just merge this as-is

**Comment by m-chaturvedi** [comment]
@mattdawkins I'm sorry I forgot to comment, but I did try to add support for dynamic ports

**Pull request merged**